ui/src/plugins/dev.perfetto.TrackEvent/index.ts

```diff
@@ -141,6 +144,10 @@ export default class implements PerfettoPlugin {
             uri,
             {
               unit: unit ?? undefined,
+              yRangeSharingKey:
+                yAxisShareKey === null
+                  ? undefined
+                  : `${yAxisShareKey}-${parentId}`,
```

Comment:
This should eb changed to something like `trackEvent:parent-${parentId}-key-${yAxisShareKey}